### PR TITLE
[NEW] First interface. Tasks displayed + Comment modification

### PR DESCRIPTION
**Pull request by LedruRollin**
This is a first basic interface which dialogs with the local gazu API.
It enables a user to see all its to-do tasks, and leave a comment thanks to a button.

### CODE REVIEW

File: gazupublisher/views/TasksTab.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    else:
+                        item = QtWidgets.QTableWidgetItem(task[col])
+                else:
+                    item = QtWidgets.QTableWidgetItem()
+                item.setTextAlignment(4)
+                item.setFlags(QtCore.Qt.ItemIsEditable and QtCore.Qt.ItemIsEnabled)
+
+                self.setItem(nb_row, nb_col, item)
+
+            # Add buttons to post comment
```
Comment (frankrousseau) [inline gazupublisher/views/TasksTab.py]: Using comments inside a function means that you should split your function in several functions.

File: gazupublisher/views/TasksTab.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            self.tasks_to_do = gazu.user.all_tasks_to_do()
+        except:
+            raise ConnectionError("Could not get user tasks")
+
+        for nb_row, task in enumerate(self.tasks_to_do):
+            current_row_nb = self.rowCount() + 1
+            self.setRowCount(current_row_nb)
+            for nb_col, col in enumerate(self.list_ids):
+                assert(col in task)
+                if task[col] != {}:
```
Comment (frankrousseau) [inline gazupublisher/views/TasksTab.py]: I think you could extract this "rendering" step inside a dedicated function.

File: gazupublisher/views/TasksTab.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                    if col == "last_comment":
+                        item = QtWidgets.QTableWidgetItem(task[col]["text"])
+                    else:
+                        item = QtWidgets.QTableWidgetItem(task[col])
+                else:
+                    item = QtWidgets.QTableWidgetItem()
+                item.setTextAlignment(4)
+                item.setFlags(QtCore.Qt.ItemIsEditable and QtCore.Qt.ItemIsEnabled)
```
Comment (frankrousseau) [inline gazupublisher/views/TasksTab.py]: I would say that this table is not editable.

File: gazupublisher/views/CommentWindow.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+        self.le = QtWidgets.QTextEdit(self)
+        self.le.setFixedSize(290, 120)
+        # self.le.move(130, 22)
+
+        self.setGeometry(300, 300, 290, 150)
+        self.setWindowTitle('Comment')
+        self.show()
+
+    def sendComment(self):
```
[the remark this answers is not in the document]
Comment (LedruRollin) [reply gazupublisher/views/CommentWindow.py]: What do you mean by "status of the comment" ? Is it the status of the task ?

[the remark this answers is not in the document]
Comment (frankrousseau) [reply gazupublisher/views/CommentWindow.py]: Yes, sorry, that is what I meant.

[the remark this answers is not in the document]
Comment (LedruRollin) [reply gazupublisher/views/CommentWindow.py]: Up @frankrousseau

**Pull request closed without merging**